RELEASES.md

```diff
@@ -0,0 +1,248 @@
+# Versioning and Release
+
+This document details the versioning and release plan for containerd. Stability
+is a top goal for this project and we hope that this document and the processes
+it entails will help to achieve that. It covers the release process, versioning
+numbering, backporting, API stability and support horizons.
+
+If you rely on containerd, it would be good to spend time understanding the
+areas of the API that are and are not supported and how they impact your
+project in the future.
+
+This document will be considered a living document. Supported timelines,
+backport targets and API stability guarantees will be updated here as they
+change.
+
+If there is something that you require or this document leaves out, please
+reach out by [filing an issue](https://github.com/containerd/containerd/issues).
+
+## Releases
+
+Releases of containerd will be versioned using dotted triples, similar to
+[Semantic Version](http://semver.org/). For the purposes of this document, we
+will refer to the respective components of this triple as
+`<major>.<minor>.<patch>`. The version number may have additional information,
+such as alpha, beta and release candidate qualifications. Such releases will be
+considered "pre-releases".
+
+### Major and Minor Releases
+
+Major and minor releases of containerd will be made from master. Releases of
+containerd will be marked with GPG signed tags and announced at
```

Comment:
It would be good to reference the fingerprint of the signing key(s) here (and more generally to get it into the WoT or at least widely distributed).